### ISSUE TITLE
Freetext import -> Similar Attributes correlates on "port" instead of "ip" - "ip:port" field

### ISSUE THREAD

**Issue by raw-data** [MISP/MISP#6050]
When manually adding a relative high number of IOCs (~7000)  via the (Web GUI) _freetext import_, the front end app keeps loading for few minutes and returns an error message " An Internal Error Has Occurred"

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Debian (9.12)
| PHP version               |  7.0
| MISP version / git hash   | 2.4.128

### Expected behavior
IOCs should be loaded as usual, before the upgrade to 2.4.128 (starting version was 2.4.123) this behaviour was never observed. Replicating the same steps on a 2.4.124 MISP instance, didn't trigger any error, IOCs gets processed by the workers and loaded inside the event.

Both systems, the one upgraded to 2.4.128, and the 2.4.124, are relative small and from the same image, i.e. vCPU: x2, RAM: 6GB.

### Actual behavior
After submitting the IOCs via the _feetext import_ option, the front end app keeps loading and the user is greeted with an "An Internal Error Has Occurred" message.

### Steps to reproduce the behaviour
1. Create a new MISP event
2. Add IOCs via the free text import option, at least ~7000

### Logs, screenshots, configuration dump, ...

```
==> tmp/logs/error.log <==
2020-06-23 17:32:21 Error: Fatal Error (1): Allowed memory size of 2147483648 bytes exhausted (tried to allocate 20480 bytes) in [/var/www/MISP/app/Model/Attribute.php, line 3457]
2020-06-23 17:32:21 Error: [InternalErrorException] Internal Server Error
Request URL: /events/freeTextImport/9033?_=1592926263528
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Allowed memory ...', '/var/www/MISP/a...', 3457)
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Allowed memory ...', '/var/www/MISP/a...', 3457, Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
```

**Comment by raw-data** [comment]
After further investigating the issue, I was able to replicate the same anomaly on a clean VM (MISP_v2.4.128@1d94458) from https://www.circl.lu/misp-images/latest/.

It looks like the problem relies on the `Similar Attributes` function (ip-dst|port / ip-src|port), that checks the port field instead of the IP address (or ip:port mapping).

## Steps to reproduce the behaviour
1. Create a MISP event with few dummy ip:port IOCs (ideally different ip with the same port, i.e. 1.1.1.1:443, 2.2.2.2:443, 3.3.3.3:8080)
2. Create another event and import (via the `freetext import` module) additional dummy ip:port. This time, different IP but same ports (i.e. 4.4.4.4:443, 5.5.5.5:443, 6.6.6.6:8080)

![snip](https://user-images.githubusercontent.com/29753218/85530533-4dd7b100-b60e-11ea-91c1-e5dec81a5c1e.png)

**Comment by raw-data** [comment]
I see only now - due to the latest tests that pointed me to the real problem - that this issue was also addressed by @elhoim in #5901 . @iglocska any workaround?

**Comment by mokaddem** [comment]
@raw-data Your issue should now be solved with https://github.com/MISP/MISP/commit/97d63a0989dc57f3da18119b656f14d56a7fa308.
Do not hesitate to get back to us if it's not the case.
